_schemaV2.graphql

```diff
@@ -2163,20 +2163,16 @@ type ArtistRelatedData {
   ): ArtistConnection
 }
 
-type ArtistSeries {
-  # List of Mongo IDs for associated Artists
+type ArtistSeries implements Node {
   artistIDs: [String!]!
-  artists(page: Int, size: Int): [Artist]
-  artworkIDs: [ID!]!
```

Comment:
This field and the below `artworksConnection` I'm dropping - I don't see any selections of these in any `ArtistSeries` fragments in clients.